Control1.1_14.04.22/SparseVector/SparseVector.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using NUnit.Framework;
+
+namespace SparseVector.Tests;
+
+public class Tests
+{
+    [Test]
+    public void VectorCreatedWithLength()
+    {
+        IVector myVector = new Vector(10);
+        Assert.IsTrue(myVector.Length == 10);
+    }
+
+    [Test]
+    public void VectorCreatedUsingArray()
+    {
+        IVector myVector = new Vector(new []{5, 3, 0, 1, 0},10);
+        Assert.Pass();
+    }
+
+    [Test]
+    public void PositionChangesAndReturns()
+    {
+        IVector myVector = new Vector(new []{5, 3, 0, 1, 0},10);
+        bool result = myVector.GetPosition(1) == 3;
```

Comment:
```suggestion
        Assert.AreEqual(3, myVector.GetPosition(1));
```
Если условие не выполнится, тест дальше не пойдёт.